tests/test_graphql.py

```diff
@@ -137,7 +137,10 @@ def test_download_graphql_asset_with_userdata(
     )
     assert response.status_code == 200, f"{response.data}"
     reply = json.loads(response.data)
+    assert len(reply["errors"]) == 1
+    # Make sure the subgraph is deployed. Response has 200 OK status code from subgraph
+    # Due to lack of CI resources, assert the specific error
     assert (
-        len(reply["data"]["nfts"]) == 1
-    )  # make sure our parametrized query works, otherwise we will get a lot of nfts
-    assert reply["data"]["nfts"][0]["id"] == asset.nftAddress.lower()
+        reply["errors"][0]["message"]
```

Comment:
wait, why? We should be able to retrieve assets from subgraph. Why are we asserting that subgraph is not deployed?

Reply:
This is the only test that fails due to the fact that I have added in the flow "--skip-subgraph-deploy". that's why I wanted to assert that specific order. The CI does not support c2d, subgraph, RBAC all in one workflow. To reduce the space, I chose to skip the subgraph deployment. If I completely remove subgraph from the workflow, multiple tests from graphql fail. So, I have decided to keep subgraph and assert the specific error.

Reply:
I can delete the assert of the message (if needed), but I just wanted to highlight the error first just so we can be aware of it when we skip subgraph deployment.

cc @alexcos20

Reply:
done by adding a new subgraph query